### DISCUSSION TITLE
Latence

### DISCUSSION THREAD

**Discussion by BenClam** [GrandOrgue/grandorgue#1078] [Help]
J'ai vérifié ma version
c'est la dernière 0.3.1.2330
J'ai toujours le problème de latence

**Comment by oleg68** [comment]
La derniere est [3.6.3](https://github.com/GrandOrgue/grandorgue/releases/tag/3.6.3-1)

@BenClam  S'il vous plait, 
1. Mettez la version de GrandOrgue a jour
2. Postez la capture d'ecran de Settings->Audio ici
